scripts/contentScript.js

```diff
@@ -146,13 +148,20 @@ async function getTrackedRepos(orgName, userId, repoHost) {
 	let url = ''
 	switch (repoHost) {
 		case 'github':
-			body = { org: orgName, userId: userId, is_github: true }
+			body = { org: orgName, userId: userId, repo_provider: 'github' }
```

Comment:
These changes are sensitive and should have been part of a different PR. 
That PR would have to be pushed with a simultaneous release in the backend as well.

@tapishr Can we even do this now in a non-backwards compatible way, that we have users on our platform?

Reply:
We can add compatibility in backend. @rhydberg - Can you add code so that it works for both is_provider, as well as repo_provider at api endpoints in devprofiler-cloudrun?

Reply:
^ `is_github`. But yes, that would be fine.